src/org/usfirst/frc/team2335/robot/OperatorInterface.java

```diff
@@ -1,52 +1,99 @@
 package org.usfirst.frc.team2335.robot;
 
+import org.usfirst.frc.team2335.robot.commands.AutoFromStation1;
 import org.usfirst.frc.team2335.robot.commands.CenterRobot;
+import org.usfirst.frc.team2335.robot.commands.IntakeBalls;
+import org.usfirst.frc.team2335.robot.commands.Lift;
 import org.usfirst.frc.team2335.robot.commands.PositionRobot;
+import org.usfirst.frc.team2335.robot.commands.ShootGroup;
 import org.usfirst.frc.team2335.robot.commands.Strafe;
 import org.usfirst.frc.team2335.robot.triggers.Axis;
 
 import edu.wpi.first.wpilibj.Joystick;
 import edu.wpi.first.wpilibj.buttons.JoystickButton;
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 
-//TODO: Rename to OperatorInterface after basic functionality for clarity.
-//This class connects operator control to different commands
-public class OI
+public class OperatorInterface
 {	
 	Joystick controller, xbox;
+	
+	//Auto test buttons
+	JoystickButton autoTest;
+	
+	//Climb buttons
+	JoystickButton climb;
+	
+	//Shoot buttons
+	JoystickButton shootBall, intakeBalls;
+	
+	//Vision buttons
 	JoystickButton center, position;
 	Axis strafe;
 	
 	/**
 	 * OperatorInterface constructor. Always initialize last in robot init.
-	 * Otherwise, it will crash since TODO: FILL-IN REASON.
```

Comment:
"Otherwise, it will crash since TODO: FILL-IN REASON"
Could you please explain the reason?